jsk_interactive_markers/jsk_interactive_marker/src/marker_6dof.cpp

```diff
@@ -255,7 +278,9 @@ class Marker6DOF {
     pose.header = feedback->header;
     pose.pose = feedback->pose;
     latest_pose_ = pose;
-    pose_pub_.publish(pose);
+    if (!publish_pose_periodically_) {
```

Comment:
here is called only when marker is moved via Rviz. So publish function here should be enabled only when publish_pose_periodically_ is "not" true.
publish_pose_periodically_ is false in default.